README.md

```diff
@@ -159,26 +117,41 @@ w.create()
 
 ## Presentations
 
+<!-- Add 3 most recent talks here -->
+
 - [KubeCon/ArgoCon NA 2024 - Data Science Workflows Made Easy: Python-Powered Argo for Your Organization](https://www.youtube.com/watch?v=hZOcj5uVQOo&list=PLj6h78yzYM2Ow7Jy0paxwrimeuFGONU_7&index=14)
 - [KubeCon/ArgoCon EU 2024 - Orchestrating Python Functions Natively in Argo Using Hera](https://www.youtube.com/watch?v=4G3Q6VMBvfI&list=PLj6h78yzYM2NA4NbSC6_mQNza2r3WV87h&index=4)
 - [CNCF TAG App-Delivery @ KubeCon NA 2023 - Automating the Deployment of Data Workloads to Kubernetes with ArgoCD, Argo Workflows, and Hera](https://www.youtube.com/watch?v=NZCmYRVziGY&t=12481s&ab_channel=CNCFTAGAppDelivery)
+
+<details><summary><i>More presentations</i></summary>
+
 - [KubeCon/ArgoCon NA 2023 - How to Train an LLM with Argo Workflows and Hera](https://www.youtube.com/watch?v=nRYf3GkKpss&ab_channel=CNCF%5BCloudNativeComputingFoundation%5D)
     - [Featured code](https://github.com/flaviuvadan/kubecon_na_23_llama2_finetune)
 - [KubeCon/ArgoCon EU 2023 - Scaling gene therapy with Argo Workflows and Hera](https://www.youtube.com/watch?v=h2TEw8kd1Ds)
 - [DoKC Town Hall #2 - Unsticking ourselves from Glue - Migrating PayIt's Data Pipelines to Argo Workflows and Hera](https://youtu.be/sSLFVIIEKcE?t=2088)
 - [Argo Workflows and Events Community Meeting 15 June 2022 - Hera project update](https://youtu.be/sdkBDPOdQ-g?t=231)
 - [Argo Workflows and Events Community Meeting 20 Oct 2021 - Hera introductory presentation](https://youtu.be/QETfzfVV-GY?t=181)
 
+</details>
+
 ## Blogs
 
+<!-- Add 3 most recent blogs here -->
+<!-- Currently 4 blogs - add collapsable section when next blog is added, and remove this comment -->
+
 - [How To Get the Most out of Hera for Data Science](https://pipekit.io/blog/how-to-get-the-most-out-of-hera-for-data-science)
 - [Data Validation with Great Expectations and Argo Workflows](https://towardsdatascience.com/data-validation-with-great-expectations-and-argo-workflows-b8e3e2da2fcc)
 - [Hera introduction and motivation](https://www.dynotx.com/hera-the-missing-argo-workflows-python-sdk/)
 - [Dyno is scaling gene therapy research with cloud-native tools like Argo Workflows and Hera](https://www.dynotx.com/argo-workflows-hera/)
 
 ## Contributing
 
-See the [contributing guide](./CONTRIBUTING.md)!
+Use one of the following to open the repo in a cloud dev box:
+
+<a href="https://codespaces.new/argoproj-labs/hera"><img src=https://github.com/codespaces/badge.svg height="40" alt="Open in GitHub Codespaces"></a>
```

Comment:
this looks weirdly big, maybe a bit smaller + specify the height in `ex`s to scale with the font?

![image](https://github.com/user-attachments/assets/1a0cdaba-73c8-466c-aa48-b04b0fea95c4)

Reply:
Yes annoyingly the buttons at their native sizes are different, and style options are limited with github markdown. I'll see if `ex` works

Reply:
>CSS units (like em or rem) are not directly supported in the height attribute.

https://www.geeksforgeeks.org/html-height-attribute/

And `style` is not supported in GitHub markdown 😭